mozperftest_tools/setup.py

```diff
@@ -0,0 +1,36 @@
+import setuptools
+
+PACKAGE_NAME = "mozperftest_tools"
+PACKAGE_VERSION = "0.1.0.dev9"
+
+# dependencies
+deps = ["numpy >=1.20.2", "matplotlib >=3.4.1,<3.5", "opencv-python", "scipy", "requests"]
+
+with open("README.md", "r", encoding="utf-8") as fh:
+    long_description = fh.read()
+
+setuptools.setup(
+    name=PACKAGE_NAME,
+    version=PACKAGE_VERSION,
+    description=("This repository is a collection of various tools that are useful for"
+                "the things we do in Performance and Performance Testing. "),
+    long_description=long_description,
+    long_description_content_type="text/markdown",
+    url="https://github.com/pypa/mozperftest-tools",
+    project_urls={
+        "Bug Tracker": "https://github.com/pypa/mozperftest-tools/issues",
+    },
+    classifiers=[
+        "Programming Language :: Python :: 3",
+        "License :: OSI Approved :: MIT License",
+        "Operating System :: OS Independent",
+    ],
+    # Get strings from http://pypi.python.org/pypi?%3Aaction=list_classifiers
+    keywords="mozilla",
+    author="Firefox Performance Test Engineering team",
+    author_email="perftest@mozilla.com",
+    package_dir={"mozperftest_tools": "mozperftest_tools"},
+    install_requires=deps,
+    packages=setuptools.find_packages(where="."),
+    python_requires=">=3.6,<3.10",
```

Comment:
I ran `pip install mozperftest-tools` and it said the I needed to have python 3.8 but python 3.6/3.7 should also work.